web/src/main/java/com/marklogic/hub/web/model/HubSettings.java

```diff
@@ -238,73 +202,63 @@ public void setProjectDir(String projectDir) {
         this.projectDir = projectDir;
     }
 
-    @Value("${mlHost}")
-    String host = null;
-
-    String name = "data-hub";
-
-    @Value("${mlStagingDbName}")
-    String stagingDbName = null;
-
-    @Value("${mlStagingTriggersDbName}")
-    String stagingTriggersDbName = null;
-
-    @Value("${mlStagingSchemasDbName}")
-    String stagingSchemasDbName = null;
-
-    @Value("${mlStagingAppserverName}")
-    String stagingHttpName = null;
-
-    @Value("${mlStagingForestsPerHost}")
-    Integer stagingForestsPerHost = null;
-
-    @Value("${mlStagingPort}")
-    Integer stagingPort = null;
-
-    @Value("${mlStagingAuth}")
-    String stagingAuthMethod = null;
+    public HubSettings() {
+        HubConfig config = new HubConfigImpl();
```

Comment:
This just gets its default property values from a new HubConfigImpl instead of dhf-defaults.properties.